### DISCUSSION TITLE
P4TOLINO06 Muito obrigado por disponibilizar estes mods, sensacional.

### DISCUSSION THREAD

**Discussion by evandromalmsteen** [P4TOLINO06/FSR3.0-Mod-Setup-Utility#6] [General]
@P4TOLINO06 

Muito obrigado por disponibilizar estes mods, sensacional. Me ajudou muito a rodar jogos pesados etc, valeu!

**Comment by P4TOLINO06** [comment]
Opa, tmj :)

Em ter., 8 de out. de 2024 às 00:05, evandromalmsteen <
***@***.***> escreveu:

**Reply by evandromalmsteen** [reply]
Eaee, tudo firme?!

Você vai parar de postar a versão, FSR3.0-Mod-Setup-Utility que parou na v3.2.17? Vai ser só a FSR3 Mod-Setup Utility Enhanced daqui pra frente? Eu uso o FSR3.0-Mod-Setup-Utility por conta do erro abaixo que ocorre ao executar apenas o FSR3 Mod-Setup Utility Enhanced. Mas tranquilo, só pra saber mesmo, dá pra copiar os mods pelas pastas e configurar em cada jogo.

<# Windows 11 24H2
Consulte o final desta mensagem para obter detalhes sobre como chamar a 
depuração just-in-time (JIT) em vez desta caixa de diálogo.

************** Texto de Exceção **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at FSR3ModSetupUtilityEnhanced.formSettings.formSettings_Load(Object sender, EventArgs e) in D:\New folder\FSR3ModSetupUtilityEnhanced\FSR3ModSetupUtilityEnhanced\FSR3ModSetupUtilityEnhanced\formSettings.cs:line 1330
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at InvokeStub_SendOrPostCallback.Invoke(Object, Span`1)
   at System.Reflection.MethodBaseInvoker.InvokeWithOneArg(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)


************** Assemblies Carregados **************
System.Private.CoreLib
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Private.CoreLib.dll
----------------------------------------
FSR3ModSetupUtilityEnhanced
    Versão do Assembly: 1.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\FSR3ModSetupUtilityEnhanced.dll
----------------------------------------
System.Runtime
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Runtime.dll
----------------------------------------
System.Windows.Forms
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Windows.Forms.dll
----------------------------------------
System.Windows.Forms.Primitives
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Windows.Forms.Primitives.dll
----------------------------------------
System.ComponentModel.Primitives
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.ComponentModel.Primitives.dll
----------------------------------------
System.Drawing.Primitives
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Drawing.Primitives.dll
----------------------------------------
System.Collections.Specialized
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Collections.Specialized.dll
----------------------------------------
System.Drawing.Common
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Drawing.Common.dll
----------------------------------------
System.Runtime.InteropServices
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Runtime.InteropServices.dll
----------------------------------------
System.Threading
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Threading.dll
----------------------------------------
System.Diagnostics.TraceSource
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Diagnostics.TraceSource.dll
----------------------------------------
System.Collections
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Collections.dll
----------------------------------------
System.Linq
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Linq.dll
----------------------------------------
Microsoft.Win32.Primitives
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\Microsoft.Win32.Primitives.dll
----------------------------------------
System.ComponentModel.EventBasedAsync
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.ComponentModel.EventBasedAsync.dll
----------------------------------------
System.Threading.Thread
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Threading.Thread.dll
----------------------------------------
Accessibility
    Versão do Assembly: 4.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\Accessibility.dll
----------------------------------------
System.ComponentModel.TypeConverter
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.ComponentModel.TypeConverter.dll
----------------------------------------
System.ComponentModel
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.ComponentModel.dll
----------------------------------------
System.Numerics.Vectors
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Numerics.Vectors.dll
----------------------------------------
System.Memory
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Memory.dll
----------------------------------------
System.Resources.Extensions
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Resources.Extensions.dll
----------------------------------------
System.Drawing
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Drawing.dll
----------------------------------------
System.Collections.Concurrent
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Collections.Concurrent.dll
----------------------------------------
System.ObjectModel
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.ObjectModel.dll
----------------------------------------
System.IO.FileSystem.DriveInfo
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.IO.FileSystem.DriveInfo.dll
----------------------------------------
System.Text.RegularExpressions
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Text.RegularExpressions.dll
----------------------------------------
Microsoft.Win32.SystemEvents
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\Microsoft.Win32.SystemEvents.dll
----------------------------------------
System.Collections.NonGeneric
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Collections.NonGeneric.dll
----------------------------------------
System.Text.Json
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Text.Json.dll
----------------------------------------
System.Net.Http
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Net.Http.dll
----------------------------------------
System.Net.Primitives
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Net.Primitives.dll
----------------------------------------
System.Diagnostics.Tracing
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Diagnostics.Tracing.dll
----------------------------------------
System.Diagnostics.DiagnosticSource
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Diagnostics.DiagnosticSource.dll
----------------------------------------
System.Net.Security
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Net.Security.dll
----------------------------------------
System.Security.Cryptography
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Security.Cryptography.dll
----------------------------------------
System.Private.Uri
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Private.Uri.dll
----------------------------------------
System.Net.Sockets
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Net.Sockets.dll
----------------------------------------
System.Threading.Overlapped
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Threading.Overlapped.dll
----------------------------------------
System.Net.NameResolution
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Net.NameResolution.dll
----------------------------------------
System.Threading.ThreadPool
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Threading.ThreadPool.dll
----------------------------------------
System.Diagnostics.Process
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Diagnostics.Process.dll
----------------------------------------
System.Runtime.Intrinsics
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Runtime.Intrinsics.dll
----------------------------------------
System.Text.Encoding.Extensions
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Text.Encoding.Extensions.dll
----------------------------------------
System.Security.Principal.Windows
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Security.Principal.Windows.dll
----------------------------------------
System.Security.Claims
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Security.Claims.dll
----------------------------------------
System.Runtime.Loader
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Runtime.Loader.dll
----------------------------------------
System.Windows.Forms.resources
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\pt-BR\System.Windows.Forms.resources.dll
----------------------------------------
System.Diagnostics.StackTrace
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Diagnostics.StackTrace.dll
----------------------------------------
System.Reflection.Metadata
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Reflection.Metadata.dll
----------------------------------------
System.Collections.Immutable
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Collections.Immutable.dll
----------------------------------------
System.IO.MemoryMappedFiles
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.IO.MemoryMappedFiles.dll
----------------------------------------
System.Text.Encodings.Web
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Text.Encodings.Web.dll
----------------------------------------
System.Reflection.Emit.Lightweight
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Reflection.Emit.Lightweight.dll
----------------------------------------
System.Reflection.Primitives
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Reflection.Primitives.dll
----------------------------------------
System.Reflection.Emit.ILGeneration
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Reflection.Emit.ILGeneration.dll
----------------------------------------
Microsoft.Web.WebView2.WinForms
    Versão do Assembly: 1.0.3485.44
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\Microsoft.Web.WebView2.WinForms.dll
----------------------------------------
Microsoft.Web.WebView2.Core
    Versão do Assembly: 1.0.3485.44
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\Microsoft.Web.WebView2.Core.dll
----------------------------------------
mscorlib
    Versão do Assembly: 4.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\mscorlib.dll
----------------------------------------
System.Diagnostics.FileVersionInfo
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Diagnostics.FileVersionInfo.dll
----------------------------------------
System.Threading.Tasks
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Threading.Tasks.dll
----------------------------------------
System.Runtime.Extensions
    Versão do Assembly: 8.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.Runtime.Extensions.dll
----------------------------------------
System
    Versão do Assembly: 4.0.0.0
    Localização: G:\Mod FSR 3\FSR3 Mod Setup Utility\FSR3 v5.3\System.dll
----------------------------------------

************** Depuração JIT **************

#>

Abs

**Comment by P4TOLINO06** [comment]
Opa

O FSR3.0-Mod-Setup-Utility está passando por uma refatoração e já está
quase concluído. Falta apenas a parte visual, e a versão 4.0 deve ser
lançada em cerca de duas semanas. :)

Em sáb., 13 de set. de 2025 às 11:06, evandromalmsteen <
***@***.***> escreveu:

**Reply by evandromalmsteen** [reply]
Ah legal, saindo eu irei baixá-lo.

Valeu ae man, obrigado!

Abs
